src/databrowser.js

```diff
@@ -165,7 +165,7 @@ define([
                 }
                 info = info._parent;
             }
-            vellum.handleDropFinish(target, path);
+            vellum.handleDropFinish(target, path, widget.mug);
```

Comment:
I should've linked to my dev branch instead of master: https://github.com/dimagi/Vellum/blob/19f7ce7895fdf928ec7a4df4fc14cd371de5ecd6/src/core.js#L736

That trickles down to:
https://github.com/dimagi/Vellum/blob/19f7ce7895fdf928ec7a4df4fc14cd371de5ecd6/src/widgets.js#L741

If ES6 was supported I'd use default params to make the code a little cleaner, but this was a an easy way to make case properties drag/droppable with bubbles.

My next PR is to refactor/add tests to some of the current functionality and I'll change the involved functions to not rely on form